analyses/transcriptomic-dimension-reduction/util/dimension-reduction-functions.R

```diff
@@ -12,7 +12,7 @@ perform_dimension_reduction <- function(transposed_expression_matrix,
   # Given a data.frame that containes the transposed expression matrix and the
   # name of a dimension reduction method, perform the dimension reduction
   # technique on the transposed matrix
-  # 
+  #
   # Args:
   #   transposed_expression_matrix: Transposed `RSEM` or `Kallisto` expression
   #                                 matrix
```

Comment:
I can't comment on it directly because it's not a part of the current PR, but the comment says that seed is set to a default of 2019 but this function has no default set.